sdk/monitor/ci.yml

```diff
@@ -47,7 +47,7 @@ extends:
       safeName: AzureMonitorQuery
     - name: Azure.Monitor.Ingestion
       safeName: AzureMonitorIngestion
-    CheckAOTCompat: true
-    AOTTestInputs:
-    - ArtifactName: Azure.Monitor.OpenTelemetry.Exporter
-      ExpectedWarningsFilepath: None
+    # CheckAOTCompat: true
```

Comment:
is it necessary to comment out along with warnings file?
Also, it seems exporter was AOT-compatible and now it's not - is this ok?

Reply:
> is it necessary to comment out along with warnings file?

I'm having blockers with the warnings file. I worked with Maddy on Friday and together we couldn't get the formatting correct. We decided to disable this check to unblock work and keep working on the exceptions file this week.

> is this ok?

There's going to be a regression as the LiveMetrics code is not AOT-compatible. Our next release will be a beta and we decided this is acceptable. We'll continue working towards full compatibility.